TakingITGlobal/commit2act
# [TIG-810] Env var for log action with routes and placeholder page

## Ticket

* [TIG-810](https://app.clickup.com/t/9009201449/TIG-810)

## Description

This PR takes care of a few preliminary elements for the improvements to Log Action:

* Adds `REACT_APP_LOG_ACTION_STEPS` environment variable
* Adds `.env.development` file and start script with `NODE_ENV` set to dev
* Adds `isSimpleLogActionSteps` utility to check for env var
* Sets up granular Log Action paths object and routing array
* Conditionally renders routes within `Routing` handler
* Adds basic `LogActionPlaceholder` page component with sample nav and output for `actionType` and `actionId` params

## Testing

1. Pull down and install the branch (`npm install`)
2. Run the local app in development mode (`npm run start:dev`)
3. Log in and select "Log Action" either from the main nav or the Dashboard prompt
4. Confirm the first page renders a "Log Action" heading with no type or ID indicated
5. Use the links within the page to navigate throughout steps
6. Confirm steps 2-3 display an action type
7. Confirm steps 4-6 display an action type and id

## Screenshots

**First page with placeholder links and values:**

![image](https://github.com/TakingITGlobal/commit2act/assets/1672105/ae6e199d-a548-4688-99bb-a11940ba3c51)

**Final page with placeholder values populated:**

![image](https://github.com/TakingITGlobal/commit2act/assets/1672105/ea0ddda6-09db-4a1d-a0e8-f5d8fb89ef6b)
